mf2py/dom_helpers.py

```diff
@@ -176,6 +176,8 @@ def text_collection(el, replace_img=False, img_to_src=True, base_url=''):
     for t in results:
         if t in (P_BREAK_BEFORE, P_BREAK_AFTER):
             count = max(t, count)
+        elif t == PRE_BEFORE:
+            text = text.rstrip(' ')
         elif not isinstance(t, int):
             if count or last == '\n':
                 t = re.sub(r'^ +', '', t)
```

Comment:
`t.lstrip(' ')` instead of regex?

Reply:
yes! good call. i actually had that originally, then switched for some reason i've now forgotten. 😁 no matter, will do.